rpmio/rpmpgp_internal.c

```diff
@@ -572,28 +572,28 @@ static int pgpPrtPubkeyParams(uint8_t pubkey_algo,
 		const uint8_t *p, const uint8_t *h, size_t hlen,
 		pgpDigParams keyp)
 {
-    int rc = 1;
+    int rc = 1; /* assume failure */
     const uint8_t *pend = h + hlen;
     int curve = 0;
+    if (keyp->tag != PGPTAG_PUBLIC_KEY && keyp->tag != PGPTAG_PUBLIC_SUBKEY)
+	return rc; /* Not a public key */
```

Comment:
This isn't in rpm style of commenting: it's stating the obvious, and code comments should be on their own lines unless there's a very, very good reason. This doesn't. Variable declarations are the one common exception to the rule.

As this test repeats elsewhere, I'd suggest adding a (private) helper function/macro, something like isKey().